src/server/views/transition-page.ejs

```diff
@@ -0,0 +1,139 @@
+                text-align: center;
+            }
+
+            #bottom-container {
+                height: 50vh;
+                background-color: #384a51;
+                text-align: center;
+                position: relative;
+            }
+
+            #bottom-container img {
+                position: absolute;
+                bottom: 0;
+                margin-bottom: 110px;
+                width: 96px;
+                height: 30px;
+                margin-left: -48px;
+            }
+
+            #main-message {
+                display: inline-block;
+                margin-top: 15vh;
+                height: auto;
+                width: 400px;
+            }
+
+            #main-message p {
+                line-height: 20px;
+                font-size: 16px;
+            }
+
+            #main-message h2 {
+                font-weight: 500;
+                line-height: 20px;
+                font-size: 28px;
+            }
+
+			#loader-circle {
+				margin: 42px 0px;
+			}
+
+			#loader-circle img {
+				animation: rotation 3s linear infinite;
+				width: 24px;
+				height: 24px;
+			}
+
+            #card {
+                display: block;
+                position: absolute;
+                top: 50%;
+                left: 50%;
+                margin-left: -286px;
+                margin-top: -60px;
+                width: 500px;
+                height: 120px;
+                text-align: center;
+                background-color: white;
+                border-radius: 10px;
+                box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.1);
+                padding: 24px 36px;
+                z-index: 1000;
+            }
+
+            #card h3 {
+                font-size: 12px;
+                line-height: 1.5;
+                color: #384a51;
+            }
+
+            #card p {
+                font-size: 14px;
+                line-height: 1.43;
+                font-weight: 200;
+            }
+
+			@keyframes rotation {
+				from {
+					transform: rotate(0deg);
+				}
+
+				to {
+					transform: rotate(359deg);
+				}
+			}
+
+        </style>
+    </head>
+
+    <body>
+        <div id="top-container">
+            <div id="main-message">
+                <p>Check your browser to make sure you're visiting</p>
+                <h2><b>go.gov.sg/<%= shortUrl _%></b></h2>
+            </div>
+			<!-- loader circle -->
+			<div id="loader-circle">
+				<img src="https://gosg-public.s3-ap-southeast-1.amazonaws.com/icon-loader.svg"/>
+			</div>
+            <div id="card">
+                <h3>You will be redirected to your requested content in a few seconds:</h3>
+                <p><%= longUrl _%></p>
+            </div>
+
+        </div>
+        <div id="bottom-container">
+            <img src="https://gosg-public.s3-ap-southeast-1.amazonaws.com/go-logo.svg"/>
+        </div>
+        <script>
+            setTimeout(function() {
+                window.location = '<%- longUrl %>'
+            }, 3000)
```

Comment:
Should we also template out the duration before redirecting to `redirect.ts`?

Reply:
potentially, yeah it could be an optional environment variable